.yarnrc.yml

```diff
@@ -0,0 +1,9 @@
+nodeLinker: node-modules
```

Comment:
It matches the behaviour of Yarn 1 by using a `node_modules` folder for dependencies, instead of Yarn PnP or pnpm.

Reply:
Ensures that we use `node_modules` instead of the Plug N' Play (`pnp`) installation strategy: https://yarnpkg.com/getting-started/migration/#step-by-step

Reply:
I've found (from experimenting with converting the controllers repo to a monorepo) that this is necessary when using TypeScript, as TypeScript is unable to resolve types that are provided via `@types/*` packages. Was this the case as well?

Reply:
It was the default value when upgrading, and it seems like it's compatible with most tools, so I didn't touch it.